marvell.py

```diff
@@ -1,12 +1,22 @@
 import os
 import shlex
+import typing
 from clustersConfig import NodeConfig
+from bmc import BMC
 from bmc import BmcConfig
 import common
 import host
+from logger import logger
+import coreosBuilder
+from nfs import NFS
 
 
-def marvell_bmc_rsh(bmc: BmcConfig) -> host.Host:
```

Comment:
Derive from BMC just like BMCBF. Most of what you need is already in bmcbf.py. Can we uncomment that, make it more generic, and start using it here?

Reply:
Much better. A few more comments below.